frontend/webapp/views/main/templates/hijack.htm

```diff
@@ -18,11 +18,6 @@
         <div class="row">
             <div class="col-lg-12">
                 <h1 class="page-header">Viewing Hijack <small id="hijack_status"></small></h1>
-                {% if mitigate == True and configured != True %}
```

Comment:
unnecessary any more (it is always configured)

Reply:
@pgigis again here, I removed the check since I think it is not needed anymore, correct?